src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -2218,6 +2218,7 @@ export const handleUpdateStartPage = [
         fileId: Joi.when('state', {
           is: FormLogoState.Custom,
           then: Joi.string().required(),
+          otherwise: Joi.any().forbidden(),
```

Comment:
ensure that this PR goes out with the specifc start page update PR, or it won't be backwards compatible

Reply:
yeap, the base branch for this PR is the `start-page` branch, will merge this into the dependent branch before finally merging into develop